clang/test/dpct/sharedmem-dim-noUSM.cu

```diff
@@ -83,9 +67,7 @@ __global__ void kernel3() {
 }
 
 void foo3() {
-  // CHECK: /*
-  // CHECK-NEXT: DPCT1083:{{[0-9]+}}: The size of local memory in the migrated code may be different from the original code. Check that the allocated memory size in the migrated code is correct.
-  // CHECK-NEXT: */
-  // CHECK-NEXT: sycl::local_accessor<int, 1> a_acc_ct1(sycl::range<1>(sizeof(sycl::float3) * 3), cgh);
+  // CHECK: DPCT1083:{{[0-9]+}}: The size of local memory in the migrated code may be different from the original code. Check that the allocated memory size in the migrated code is correct.
+  // CHECK: sycl::local_accessor<int, 1> a_acc_ct1(sycl::range<1>(sizeof(sycl::float3) * 3), cgh);
```

Comment:
Please double check if the changes in this file required.   Only update the test which is impacted by this PR.

Reply:
done